pkg/storaged/fsys-tab.jsx

```diff
@@ -149,71 +149,50 @@ export class FilesystemTab extends React.Component {
-                        { (old_opts)
-                            ? (
-                                <tr>
-                                    <td>{_("Mount Options")}</td>
-                                    <td>
-                                        <StorageLink onClick={mounting_dialog}>
-                                            {old_opts}
-                                        </StorageLink>
-                                    </td>
-                                </tr>
-                            )
-                            : null
-                        }
-                        { (mounted_at.length > 0 || self.props.client.is_old_udisks2)
-                            ? (
-                                <tr>
-                                    <td>{_("Mounted At")}</td>
-                                    <td>
-                                        {mounted_at.join(", ")}
-                                        <div className="tab-row-actions">
-                                            { (mounted_at.length > 0)
-                                                ? <StorageButton onClick={unmount}>{_("Unmount")}</StorageButton>
-                                                : <StorageButton onClick={mount}>{_("Mount")}</StorageButton>
-                                            }
-                                        </div>
-                                    </td>
-                                </tr>
-                            )
-                            : null
-                        }
-                        <tr>
-                            <td>{_("Used")}</td>
-                            <td>{used}</td>
-                        </tr>
-                    </tbody>
-                </table>
+                                <div className="tab-row-actions">
+                                    { !is_filesystem_mounted && <StorageButton onClick={mount}>{_("Mount")}</StorageButton> }
+                                </div>
+                            </div>
+                        </React.Fragment>
+                    )}
+
+                    { old_opts && (
+                        <React.Fragment>
+                            <label className="control-label">{_("Mount Options")}</label>
+                            <StorageLink onClick={mounting_dialog}>
+                                {old_opts}
+                            </StorageLink>
+                        </React.Fragment>
+                    )}
+
+                    { (mounted_at.length > 0 || self.props.client.is_old_udisks2) && (
+                        <React.Fragment>
+                            <label className="control-label">{_("Mounted At")}</label>
+                            <div>
+                                {mounted_at.join(", ")}
+                                <div className="tab-row-actions">
+                                    <StorageButton onClick={unmount}>{_("Unmount")}</StorageButton>
```

Comment:
This needs to offer "Mount" when mounted_at.length == 0, see the old code.

Reply:
Ah sorry, my fault. Let's remove this commit then.

Reply:
Fixed.